source/sync/get-started.txt

```diff
@@ -14,14 +14,14 @@ Get Started with {+sync+}
    :class: singlecol
 
 To sync data across devices, you :ref:`enable Device Sync <enable-sync>` for your
-{+app+} and then use the sync-related methods and properties in the SDKs.
+App and then use the sync-related methods and properties in the SDKs.
 
 .. tip:: Check out the tutorial
    
-   If you prefer to learn by example, check out the {+service-short+} 
+   If you prefer to learn by example, check out the App Services 
    `tutorial <https://www.mongodb.com/docs/realm/tutorial/>`_, which describes
    how to build a synced Task Tracker application with clients for common 
```

Comment:
```suggestion
   how to build a synced todo list application with clients for common 
```

let's rephrase. we've deprecated term 'Task Tracker'

Reply:
Okay. I've updated the few other remaining instances in the repo as well